_posts/newsletters/2022-01-31-newsletter.md

```diff
@@ -0,0 +1,205 @@
+32. [Operations Team Lead - Institute for Computational and Data Sciences (ICDS)](https://psu.wd1.myworkdayjobs.com/en-US/PSU_Staff/job/University-Park-Campus/Operations-Lead_REQ_0000020695-1): State College, Pennsylvania *Posted: Nov 09, 2021*
+33. [Researcher](https://jobs.hr.wisc.edu/en-us/job/511080/researcher): University of Wisconsin-Madison, Madison, WI *Posted: Oct 22, 2021*
+
+<a name="news"></a>
+# **Community News**
+
+* The UNESCO [Recommendation on Open Science](https://click.mailerlite.com/link/c/YT0xODM3MjgxNzE3NjYwOTQ5MDQ2JmM9bThhNCZiPTgyODY5NTY2OSZkPXczYTh2MXA=._oy_Gb9yAAS4Ino8P6O-u0yNpChfhrZlVzALpCn3SAU) has been adopted by its member states. This recommendation defines open scientific knowledge as including software and source code. In particular, the recommendation argues for users to gain free access to open source software and source code in a timely and user-friendly manner, in human- and machine-readable and modiﬁable format, under an open license.
+* The US [National Academies Roundtable on Aligning Incentives for Open Science](https://click.mailerlite.com/link/c/YT0xODM3MjgxNzE3NjYwOTQ5MDQ2JmM9bThhNCZiPTgyODY5NTc0MSZkPXM2YzJrNm4=.5O2yF7YdzjXBwF_HHdqUIuNlltbbLDpaauv910k7SGs) now incorporates an updated [good practices primer on code and software](https://click.mailerlite.com/link/c/YT0xODM3MjgxNzE3NjYwOTQ5MDQ2JmM9bThhNCZiPTgyODY5NTc0NCZkPWs2ZzdmOG4=.vXMYr7IqaGx5e-dAy2SLttrv_SOn93V12CbTK0GWWYc), that includes recommendations, resources and examples of open code and software policies.
+
+<a name="dei-mc"></a>
+# **DEI Media Club**
+
+An instance of the [DEI Media Club](https://us-rse.org/events/2022/2022-01-dei-media-club/) took place on January 31, 2022 at 12 ET/11 CT/10 MT/9 PT to discuss [Redesigning STEM education for diversity and inclusivity](https://podcasts.apple.com/ph/podcast/redesigning-stem-education-for-diversity-and-inclusivity/id1450207189?i=1000444578488). This was the last in the series of Podcast episode discussions. The next three DEI Media Clubs will discuss short videos. Join the #events or #dei-discussion channels for more updates!
+
+Want to know more or get involved with the DEI working group? Visit our [webpage](https://us-rse.org/wg/dei/) for more details!
+
+<a name="events"></a>
+# **Interesting Events**
+
+Some other events coming up may be of interest to RSEs:
+
+* [SIAM Conference on Parallel Processing for Scientific Computing (PP22)](https://www.siam.org/conferences/cm/conference/pp22) - February 23-26, 2022
+* [SEA's Improving Scientific Software Conference and Tutorials](https://sea.ucar.edu/conference/2022) - April 4-8, 2022
+* [Mini Gateways 2022](https://sciencegateways.org/minigateways2022) - April 5-7, 2022
+* [Vive la différence - research software engineers](https://www.researchsoft.org/events/2022-04/) - an in-person element from 19-22 April 2022 at the Lorentz Center in the Netherlands, and online events from March-May
```

Comment:
Element? Should that be event? or an event with an in person element? 

```suggestion
* [Vive la différence - research software engineers](https://www.researchsoft.org/events/2022-04/) - an in-person event from 19-22 April 2022 at the Lorentz Center in the Netherlands, and online events from March-May
```

Reply:
It's element. It has multiple parts. There is a piece (element) that's in person and pieces that are online over the course of two months.